libr/asm/arch/gb/gbasm.c

```diff
@@ -159,6 +166,38 @@ static bool gb_parse_ld2 (ut8 *buf, char *buf_asm) {
 	return false;
 }
 
+static bool gb_parse_ld3 (ut8 *buf, char *buf_asm) {
+	if (strlen (buf_asm) < 7) {
+		return false;
+	}
+	if (buf_asm[5] != ',') {
```

Comment:
ideally it must be valid to have any unspecified number of spaces between arguments, so expecting a comma in a specific offset is probably not a good check

Reply:
yes, that is actually valid. befor ld3 is called r_str_replace_in removes multiple spaces in line 213